javadoc/src/org/netbeans/modules/javadoc/highlighting/Highlighting.java

```diff
@@ -147,13 +154,34 @@ public void tokenHierarchyChanged(TokenHierarchyEvent evt) {
         if (seq.moveNext()) {
             int start = seq.offset();
             do {
+                String period = null;
+                int indexOfPeriod = -1;
+                for (String p : PERIODS) {
+                    int index = TokenUtilities.indexOf(seq.token().text(), p);
```

Comment:
Wouldn't it be more appropriate if Javadoc lexer returned some specific tokenID japanese period char (and ordinary period char), since the period after the 1st sentence is a lexical element. But I fear that a new token ID could break existing lexer clients.

Reply:
@sdedic So, I would like to avoid changing the JavadocLexer(java.lexer) because a new tokenID(e.g. `JavadocTokenId.JAPANESE_DOT`) is needed only for this feature at the moment. Even if someone adds the new tokenID, probably, we can notice that because I added unit tests to this feature.

However, if you prefer adding the tokenID, I'll try it :)